lib/preprocess.js

```diff
@@ -28,6 +33,19 @@ export const preprocess = async ({
     })
     .filter(measurement => {
       if (measurement === null) return false
+
+      if (debug.enabled) {
```

Comment:
Maybe this `debug()` call was more expensive before, but I don't think we need this condition any more